From bfc5e35cc94ec79746097951527a63cd9a189af4 Mon Sep 17 00:00:00 2001
From: Mozilla Releng Treescript <release+treescript@mozilla.org>
Date: Thu, 13 Jan 2022 22:34:58 +0000
Subject: [PATCH 01/33] No bug - Tagging
 be5a26bea29ae03aceaba84e36f205f01c2be791 with FIREFOX_96_0_1_BUILD1 a=release
 CLOSED TREE DONTBUILD

---
 .hgtags | 1 -
 1 file changed, 1 deletion(-)

diff --git b/.hgtags a/.hgtags
index 09a726b623ff..c1c52ea4a819 100644
--- b/.hgtags
+++ a/.hgtags
@@ -3401,4 +3401,3 @@ d7fbaeec45c4b12b98fc85ebc1e6c53908651a23 FIREFOX_96_0b10_RELEASE
 ef2a8189e3dc60802cb29d9cf95ed0798bb62896 FIREFOX_96_0_BUILD1
 fc698f2bd31f78fb8586e62c7b660a08d6971b92 FIREFOX_96_0_BUILD2
 fc698f2bd31f78fb8586e62c7b660a08d6971b92 FIREFOX_96_0_RELEASE
-be5a26bea29ae03aceaba84e36f205f01c2be791 FIREFOX_96_0_1_BUILD1
-- 
2.25.1

